src/lib.rs

```diff
@@ -410,6 +393,10 @@ impl Config {
         if self.is_static(name) {
             cmd.arg("--static");
         }
+        let target = env::var("TARGET").unwrap_or_else(|_| String::new());
+        if target.contains("msvc") {
```

Comment:
Why not
```rust
if let Some(target) = env::var("TARGET") {
    if target.contains("msvc") {
        cmd.arg("--msvc-syntax");
    }
}
```
or something with match and guards for the `contains`?
One level of indentation more, but doesn't have the empty string